Base 3061/chassis/chassisdef_blackjack_BJ-1P.json

```diff
@@ -0,0 +1,230 @@
+{
+  "Custom": {
+    "ArmActuatorSupport": {
+      "LeftLimit": "Upper",
+      "RightLimit": "Upper"
+    }
+  },
+  "Description": {
+    "Cost": 3454430,
+    "Rarity": 1,
+    "Purchasable": true,
+    "Manufacturer": "",
+    "Model": "",
+    "UIName": "Blackjack",
+    "Id": "chassisdef_blackjack_BJ-1P",
+    "Name": "Blackjack",
+    "Details": "In The Periphery Maintaining Mechs Tends To Be A Problem, As A Result The 1P was designed generally as a downgrade from The Os, But Much cheaper to maintain and Repair, Pilots often Complain About the Structure 'Making Strange Noises', The Remarks Were Usually Ignored. <b><color=#53ff1a>Restricted Actuators- Left: Upper, Right: Upper</color></b>",
```

Comment:
Entire description needs to be decapitalised except where appropriate.
 The Remarks Were Usually Ignored.
Should be "these remarks are usually ignored." because English.